pages/common/git-delete-branch.md

```diff
@@ -0,0 +1,13 @@
+# git delete-branch
+
+> Delete local and remote Git branches.
+> Part of `git-extras`. You will get errors if you are checked out on a branch you are deleting.
```

Comment:
```suggestion
> Part of `git-extras`. An error occurs on deleting the current checked out branch.
```

Reply:
```suggestion
> Part of `git-extras`. Errors will occur if you are deleting a checked out branch
```

Reply:
I agree with the removal of "you", but it seems wrong to put on the same line as "Part of `git-extras`".  I've venture to say that this is note is unnecessary, because it's not likely that a user will think that they _can_ delete the current branch.  Thoughts?

Reply:
A user might make a mistake and not realise they are checked out on a particular branch

Reply:
Well actually as I've just encountered with #5988, it does *not* give you an error if you try it...

Reply:
error: Cannot delete branch 'testremove' checked out

I have a test repo just for me and while checkedout, I ran this command and got this

> error: Cannot delete branch 'testremove' checked out

Reply:
it does however delete the remote branch since you are not _checked out_ at origin/testremove

Reply:
Just saw that I got the message too. However it deletes all remote branches. I'd prefer not to include the note above, since it may let the user think it's save to run.

Reply:
So should we resolve this thread?

Reply:
I actually think we should remove the second sentense completely. We also don't include in every Git page, that errors will occur if you're not in a Git repository.

Reply:
```suggestion
> Part of `git-extras`.
```

Reply:
But it is an easy error to make, many will make it

Reply:
How about:
```suggestion
> Part of `git-extras`. When deleting the checked out branch, only the remote branch will be deleted.
```

Reply:
```suggestion
> Part of `git-extras`. When deleting a checked out branch, only the remote branch will be deleted.
```

Reply:
Can you have multiple checked out branches?

Reply:
Not to my knowledge, however the `the` seems out of place, a fits better and makes it read better

Reply:
No, you can't have multiple checked out branches under normal operation to my knowledge.

@bl-ue: On the contrary, I think it's important to keep the `Part of git-extras.`, just in case someone confuses it for an inbuilt git command.

Reply:
I meant that the note about deleting the checked out branch isn't necessary, @sbrl.

Reply:
Can we reach a result, what should suggestion should  I apply